examples/deployment/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Use an official Python runtime as a parent image
+FROM python:3.11-slim AS build
+
+# patches & OS dependencies
+RUN apt-get update && apt-get -y upgrade && \
+    apt-get install -y git build-essential pkg-config libhdf5-dev npm && \
+    rm -rf /var/lib/apt/lists/*
+
+# python dependencies
+RUN pip3 install --no-cache-dir --upgrade pip && \
```

Comment:
Do you need to use venv?

Reply:
hmm i consider as the requirements are all in the container and isolated, there is no need to actually isolate them figure. as for our own local machine, we will need to isolate requirements from different build.

i would actually defer to your opinion on whether need to venv or not but that is my reasoning

Reply:
thanks, looks good then.